pytripgui/plan_executor/html_executor_logger.py

```diff
@@ -0,0 +1,43 @@
+
+from queue import Queue
+from pytrip.tripexecuter import ExecutorLogger
+
+
+class HtmlExecutorLogger(ExecutorLogger):
+    def __init__(self):
+        self.queue = Queue()
+
+    def info(self, text):
+        self.queue.put("<b>{}</b>".format(text))
```

Comment:
It would be nice to have those html codes in some kind of variables
Also commenting what it does would be nice for future changes